twitter_delete.rb

```diff
@@ -20,6 +20,7 @@
   opt :olds, "Keep tweets/likes more than this many days old", default: 9999
   opt :rts, "Keep tweet with this many retweets", default: 5
   opt :favs, "Keep tweets with this many likes", default: 5
+  opt :part, "Keep tweets with this many likes", default: 0
```

Comment:
This text is the same. What is this option meant to do?

Reply:
I mean to "Refer to window.YTD.tweet.part[x]" in tweet-part[x].js files. 

Refer from my tweets archive files, it's large lists of tweets get split up into multiple files, which are all found in the same place. 

So as well as "tweet.js" you might have "tweet-part1.js" (window.YTD.tweet.part1) , "tweet-part2.js" (window.YTD.tweet.part2) etc. 

Sorry I'm not edit this label before pull request to you.

Example command
./twitter_delete.rb --user Username --olds 9999 --days 28 --rts 99 --favs 99 --archive /mnt/d/tweet-part1.js --part 1

Reply:
@fordantitrust Gotcha. I think ideally this would be able to take all archive files, perhaps if passed a folder? Thoughts?